hack/jenkins/e2e.sh

```diff
@@ -164,6 +166,7 @@ GCE_SLOW_TESTS=(
 
 # Tests which are not able to be run in parallel.
 GCE_PARALLEL_SKIP_TESTS=(
+    "GCE\sL7\sLoadBalancer\sController"
```

Comment:
It's not clear to me that there is a real reason why these cannot run in parallel (other than just being a bit slow)?  Currently SLOW tests are explicitly skipped in PARALLEL runs (see line 314 in this file), so you don't need the above line, and can just omit it?